src/service/InteractiveExposeV2S3Service/InteractiveExposeV2S3Controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Entity } from '@flowfact/types';
+import { APIClient, APIMapping } from '../../http';
+
+export class InteractiveExposeV2S3Controller extends APIClient {
+    constructor() {
+        super(APIMapping.interactiveExposeV2S3Service);
+    }
+    /**
+     *
+     * returns the list of global iex V2 templates
+     *
+     */
+    getGlobalTemplateRepository() {
```

Comment:
We do not prefix any method in ff-api-services with `get`. Everything is a `fetch` because of fetching data

```suggestion
    fetchGlobalTemplateRepository() {
```

Reply:
Already done